app/presenters/error_message/message.rb

```diff
@@ -0,0 +1,61 @@
+#
+# Class to represent a single custom error message with
+# its variants of long, short and api.
+#
+module ErrorMessage
+  class Message
+    def initialize(long, short, api, key)
+      @long = long
+      @short = short
+      @api = api
+      @key = key
+    end
+
+    def long
+      substitute_submodel_numbers_and_names(@long, @key)
+    end
+
+    def short
+      substitute_submodel_numbers_and_names(@short, @key)
+    end
+
+    def api
+      substitute_submodel_numbers_and_names(@api, @key)
```

Comment:
```suggestion
      substitute_submodel_numbers_and_names(@api)
```

See below.

Reply:
https://github.com/ministryofjustice/Claim-for-Crown-Court-Defence/pull/4180#discussion_r686817255